test/runtime_tests.cc

```diff
@@ -356,6 +356,23 @@ namespace detail {
 		REQUIRE(is_safe_cgf<decltype(safe)>);
 	}
 
+	namespace foo {
```

Comment:
is this name really representative of the namespace and it functionality/purpose ?

Reply:
I would say that `foo` is indeed a good way of indicating that this is a test name without any additional purpose. I am however open to suggestions ;-).